crates/wash-lib/src/build/component.rs

```diff
@@ -416,13 +424,14 @@ async fn build_custom_component(
 }
 
 /// Generate the bindgen code that `TinyGo` components need
-async fn generate_tinygo_bindgen(user_dir: impl AsRef<Path>) -> Result<()> {
-    if !user_dir.as_ref().exists() {
+async fn generate_tinygo_bindgen(project_dir: impl AsRef<Path>) -> Result<()> {
+    if !project_dir.as_ref().exists() {
```

Comment:
```suggestion
    let project_dir = project_dir.as_ref();
    if !tokio::fs::try_exists(project_dir).await.unwrap_or_default() {
```
`try_exists` > `Path::exists()`, but since we're in `tokio` land...